src/react/components/Gallery.tsx

```diff
@@ -0,0 +1,55 @@
+import { TFile } from "obsidian";
+
+import Masonry from "~/react/masonry";
+import Card from "~/react/components/Card";
+import CardMarkdownContent from "~/react/components/CardMarkdownContent";
+import { useAppMount } from "~/react/context/app-mount-provider";
+import { getResourcePath } from "~/react/utils/render-utils";
+
+export default function Gallery({ files }: { files: TFile[] }) {
+  const { app } = useAppMount();
+  const breakpointColumnsObj = {
+    default: 10,
+    3100: 9,
+    2700: 8,
+    2300: 7,
+    1900: 6,
+    1500: 5,
+    1000: 4,
+    700: 3,
+    400: 2,
+  };
+
+  const showItems = (files: TFile[]) => {
```

Comment:
[🔧](https://github.com/erikthedeveloper/code-review-emoji-guide#:~:text=:wrench:) The name is a bit misleading here as it does not really show anything, maybe `renderFiles` would me more appropriate?